src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -280,12 +280,16 @@ private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
     GoogleProjectResource projectResource = bucketResource.getProjectResource();
     String googleProjectId = projectResource.getGoogleProjectId();
     GcsProject gcsProject = gcsProjectFactory.get(googleProjectId);
+    GoogleRegion region =
+        bucketResource.getRegion() == GoogleRegion.US
+            ? GoogleRegion.US.getRegionOrFallbackBucketRegion()
+            : bucketResource.getRegion();
```

Comment:
Since `GoogleBucketResource` is an object we control, would it make sense to move this logic there instead of putting it here?

If the code is moved, it might be good to also have it perform the `toString()` and call it something like `toLocation()` or `createLocation()`.